src/site/xdoc/server/config-jmap.xml

```diff
@@ -89,6 +89,14 @@
                     <dd>Which version of the JMAP protocol should be served when none supplied in the Accept header.
                     Defaults to draft for legacy reasons (avoid breaking changes) but setting the value to
                     rfc-8621 allow compatibility with other third party apps.</dd>
+
+                    <dt><strong>dynamic.jmap.prefix.resolution.enabled</strong></dt>
+                    <dd>Optional boolean. Defaults to false. Supported Jmap session endpoint return dynamic prefix in response.
```

Comment:
Rewrite that to my suggestions on the same text in the doc commented above please :)